src/egon/data/datasets/scenario_parameters/parameters.py

```diff
@@ -1279,6 +1319,8 @@ def heat(scenario):
             "central_gas_boiler": read_costs(
                 costs, "central gas boiler", "VOM"
             ),
+            "rural_biomass_boiler": 10,  # considering an efficiency of 0.8
+            "rural_oil_boiler": 59.7,  # considering an efficiency of 0.9
```

Comment:
Also here the source is missing

Reply:
Addressed in 7c07862fa3d78a6bc6834faaca34207b3e0f36bb